Lib/typing.py

```diff
@@ -561,23 +560,20 @@ def int_or_str(arg: int | str) -> None:
                     print("It's a str")
                 case _:
                     never_call_me(arg)  # ok, arg is of type Never
-
     """
     raise TypeError(f"{self} is not subscriptable")
 
 
 @_SpecialForm
 def Self(self, parameters):
-    """Used to spell the type of "self" in classes.
+    """Used to spell the type of "self" in classes::
```

Comment:
The first line is used alone quite frequently. For example, in IDE help text or IDE popups. Ending it with `::` might look weird.

Reply:
Good point! I'll fix that.

Reply:
```suggestion
    """Used to spell the type of "self" in classes.

    Example::
```